project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Personal pickup popup UI is broken (this seems to be a general popup problem)
<img width="695" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/ada9d728-f941-45b7-8b8d-cb86999dba94">